tools/@o3r/test-helpers/schematics/ng-add/schema.ts

```diff
@@ -0,0 +1,12 @@
+import type { JsonObject } from '@angular-devkit/core';
+
+export interface NgAddSchematicsSchema extends JsonObject {
+  /** Project name */
+  projectName: string | null;
```

Comment:
should be `projectName?: string | undefined;`

Reply:
done